server/src/source/Database/SqlLoader.cpp

```diff
@@ -338,6 +338,40 @@ std::vector<std::string> CSqlLoader::loadJobHashes(uint64_t jobId)
     return result;
 }
 
+std::vector<std::string> CSqlLoader::loadJobAllHashes(uint64_t jobId)
+{
+    std::vector<std::string> result;
+    updateSql(formatQuery("SELECT REPLACE(TO_BASE64(`hash`), '\n', '') FROM `%s` WHERE `job_id` = %" PRIu64 " ;",
```

Comment:
Please just change loadJobHashes and introduce bool load_cracked parameter. This is huge copy paste with one small difference in SQL query.

std::vector<std::string> CSqlLoader::loadJobAllHashes(bool load_cracked = false);